electron/app/components/ViewBar/ViewStage/ViewStageParameter.tsx

```diff
@@ -3,6 +3,7 @@ import { animated, useSpring } from "react-spring";
 import styled, { ThemeContext } from "styled-components";
 import { useService } from "@xstate/react";
 import AutosizeInput from "react-input-autosize";
+import { ArrowType } from "@material-ui/icons";
```

Comment:
I'm guessing this is from 91ba72bfaaccb63587f8c85cc72b9dfbbfda9c22? ArrowType isn't an actual icon - there, I defined it as either ArrowDropUp or ArrowDropDown. JSX needs custom tags to start with an uppercase letter, so it looks more like imported icon than I would like.

Reply:
Woops, fixed.